framework/draw/internal/fontsengine.cpp

```
@@ -543,81 +594,32 @@ FontsEngine::RequireFace* FontsEngine::fontFace(const Font& f, bool isSymbolMode
-    IFontFace* face = nullptr;
-    for (IFontFace* ff : m_loadedFaces) {
-        if (ff->key().dataKey == actualDataKey && ff->key().pixelSize == loadedPixelSize && ff->isSymbolMode() == isSymbolMode) {
-            face = ff;
-            break;
-        }
-    }
+    const int loadedPixelSize = loadedPixelSizeForFontPath(fontPath, requireKey.pixelSize);
 
-    //! NOTE If we haven't found a face, we'll create a new one
+    IFontFace* face = fontFace(actualDataKey, requireKey.type, loadedPixelSize, isSymbolMode);
     if (!face) {
```

Clang (14.0.6)

[warning] implicit conversion 'muse::draw::IFontFace *' -> bool

(readability-implicit-bool-conversion)